app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+
+
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str):
+        self.name = name
+
+    def visit_cafe(self, visitor: dict):
+
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError

```

Comment:
you should pass descriptive messages when you raise the exception in the visit_cafe method

Reply:
where have you read that?
![image](https://user-images.githubusercontent.com/104418771/170671862-4c3add97-f2b7-4bf6-a06b-b4145bff5a50.png)